demos/python_demos/text_detection_demo/README.md

```diff
@@ -0,0 +1,56 @@
+# Text Detection Python3 demo
+
+Decoder for Intel's version of PixelLink -- [text-detection-0001](https://github.com/opencv/open_model_zoo/blob/2019/intel_models/text-detection-0001/description/text-detection-0001.md).
+
+You will need an OpenCV compiled with Inference Engine to use this, e.g. [opencv-python-inference-engine==4.1.0.3](https://pypi.org/project/opencv-python-inference-engine/4.1.0.3/).
```

Comment:
I did it below you are looking on outdated file

Reply:
No you didn't. The demo's still not using the Inference Engine API.

Reply:
Where? It has only 31 line now:
https://github.com/banderlog/open_model_zoo/tree/develop/demos/python_demos/text_detection_demo

Are we looking on the same file?

Reply:
The README is fine. The demo itself is not. It needs to use the Inference Engine API rather than the OpenCV DNN module.

Reply:
Like `from openvino.inference_engine import IECore` and so on? If yes, why?

Reply:
> Like from openvino.inference_engine import IECore and so on?

Yes.

> If yes, why?

For consistency with the other demos.

However, I've had an internal discussion with the team about this, and we've decided to allow this demo to use the OpenCV API, as long as you rename it to `text_detection_demo_opencv` (so that users can see at a glance that it is non-standard).

Reply:
Thank you, it is good news. I renamed the demo folder and file.